tools/metashrimps/.shed.yml

```diff
@@ -0,0 +1,44 @@
+categories:
+    - Ecology
+description: Tools to operate machine actionnable jobs to EML metadata
+long_description: |
+  Tools to operate machine actionnable jobs to EML metadata from MetaShRIMPS
+name: metashrimps 
+owner: ecology 
+remote_repository_url: https://github.com/TanguyGen/metacure
+type: unrestricted
+
+repositories:

```

Comment:
can you please remove this and replace it with a suite-based shed.yml file like in your other tool

Reply:
If I am not wrong, done